src/components/Footer/Footer.scss

```diff
@@ -112,6 +112,10 @@
 .tm-w-xs {
   .wkp-footer-left {
     font-size: 11px;
+
+    .wkp-dev-portal-link {
+      display: none;
+    }
```

Comment:
The owner of the developer portal will be disappointed if we remove this link? is that ok with @mhaertwig  ?

Reply:
@mhaertwig said we can remove it on small resolutions, since the footer becomes really cramped and the content won't fit